test/utils.h

```diff
@@ -441,6 +442,11 @@ class NVFuserTest : public ::testing::Test {
     // random seed. Otherwise, use zero. If a test fails, this seed will be
     // printed.
     std::srand(getCRandomSeed());
+
+    // For the NVFuser tests, we generally want to test Fuser's matmul
+    // scheduling and lowering. Thus disabling execution via expression
+    // evaluator.
+    DisableOptionsGuard::getCurOptions().set(DisableOption::MatmulExprEval);
```

Comment:
Regardless of my other comment, we should save the previous state and recover that in TearDown.